docs-src/releases.md

```diff
@@ -0,0 +1,65 @@
+<!--
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Releases
+
+Although Service APIs are an official Kubernetes project, and represent official
+APIs, these APIs will not be installed by default on Kubernetes clusters. This
+project will use custom resource definitions (CRDs) to represent the new API
+types that Service APIs include.
+
+Similar to other Kubernetes APIs, these will go through a formal Kubernetes
+Enhancement Proposal (KEP) review. Unlike other Kubernetes APIs, Service API
+releases will be independent from Kubernetes releases.
+
+Service API releases will include four components:
+- Custom resource definitions to define the API
+- Go client libraries
+- Validation webhooks to implement cross field validations
+- Conversion webhooks to convert resources between API versions
```

Comment:
The formatting of this bulleted listis not being generated as you expect. Run `make serve` to see yourself. See https://daringfireball.net/projects/markdown/basics

Reply:
:man_facepalming: woops, thanks!